btk/sampling_functions.py

```diff
@@ -259,6 +259,82 @@ def __call__(self, table, **kwargs):
         return blend_table
 
 
+class DefaultSamplingShear(SamplingFunction):
+    """Default sampling function used for producing blend tables."""
+
+    def __init__(self, max_number=2, stamp_size=24.0, maxshift=None, shear=None, seed=DEFAULT_SEED):
```

Comment:
why not make the default (0.0, 0.0) and avoid the if statement?

Reply:
I thought about this but it may be useful if we wish to support something else than constant shear.